BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -325,8 +325,7 @@ - (NSString *)fullFolderFieldsParameterString
                        BOXAPIObjectKeyExtension,
                        BOXAPIObjectKeyIsPackage,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
-                       BOXAPIObjectKeyCollections,
-                       BOXAPIObjectKeyRepresentations];
+                       BOXAPIObjectKeyCollections];
```

Comment:
This change makes it so that we only ask for `representations` when `X-Rep-Hints` is provided and non-empty. That's what the server side API expects/supports.